packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsColorSettingCard.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled';
+
+import { ColorPickerButton } from '@/ui/input/button/components/ColorPickerButton';
+import { Card } from '@/ui/layout/card/components/Card';
+import { CardContent } from '@/ui/layout/card/components/CardContent';
+import { mainColorNames, ThemeColor } from '@/ui/theme/constants/colors';
+
+type SettingsAccountsColorSettingCardProps = {
+  onChange: (nextValue: ThemeColor) => void;
+  value: ThemeColor;
+};
+
+const StyledCardContent = styled(CardContent)`
+  display: flex;
```

Comment:
I think `display:flex` and  `justify-content `should not be passed here. It should be part of CardContent API (maybe add a props centerContent? or justifyContent = 'center' | 'left'
Same for padding I think it should be part of the API of CardContent. this will help having the same style on all usages

Overriding style from the exterior should be kept for extreme use case or positioning (margin